src/schema.rs

```diff
@@ -0,0 +1,65 @@
+use serde::{Deserialize, Serialize};
+use std::error::Error;
+use std::fs;
+
+use crate::kit_version::KitVersion;
+
+#[derive(Clone, Deserialize, Debug, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct BundleVersionSection {
```

Comment:
Question: Should the "Section" and "Schema" suffixes be dropped from these? That way they just function as domain objects rather than being named specific to the encode / decode process.